api/run.py

```diff
@@ -620,6 +621,33 @@ def getBlankForm():
     blankForm_id = ObjectId(request.json['form_id'])
     return {'data': blankFormsDOM.getFormData(blankForm_id), 'name': blankFormsDOM.getFormName(blankForm_id)}
 
+@app.route('/changeStatus', methods = ['POST'])
+@requires_auth
+@log_action('Status of form changed')
+def changeStatus():
+    form_id = ObjectId(request.json['form_id'])
+    status = request.json['form_status']
+    FormsDOM.changeCompletion(form_id,status)
+    newStatus = not status
+    print(newStatus)
+    return {'status': newStatus}
+
+@app.route('/resetForm', methods = ['POST'])
+@requires_auth
+@log_action('Form Data Reset')
+def resetForm():
+    form_id = ObjectId(request.json['form_id'])
+    newData = FormsDOM.clearForm(form_id)
+    print(newData)
+
+    newData['_id'] = str(newData['_id'])
+    newData['blank_forms_id'] = str(newData['blank_forms_id'])
+    newData['parent_id'] = str(newData['parent_id'])
+    print(newData)
+    return {'new_form_info':newData,
```

Comment:
I guess the same thing applies here -- can we just pass one thing back? Another option would be to separate everything. Just don't want duplicate data